pages/styles/portfolio.css

```diff
@@ -0,0 +1,163 @@
+#grid-container {
+    display: grid;
+    grid-template-columns: repeat(4, 300px);
+    grid-template-rows:  250px 250px;
+    justify-items: center;
+    grid-template-areas:
+    "header header header header"
+    "content content1 content2 content3"
+}
+
+#grid-container {
+    border: 5px solid #fde2f0;
+    border-radius: 5px;
+    background-color: white;
+}
```

Comment:
These two rules have exactly the same selector `#grid-container`, so they could be combined into one rule.